packages/fether-react/src/App/App.js

```diff
@@ -79,28 +133,36 @@ class App extends Component {
       <ReactResizeDetector handleHeight onResize={this.handleResize}>
         <div className='content'>
           <div className='window'>
-            <Router>
-              <Switch>
-                {/* The next line is the homepage */}
-                <Redirect exact from='/' to='/accounts' />
-                <Route path='/accounts' component={Accounts} />
-                <Route path='/onboarding' component={Onboarding} />
-                <Route path='/tokens/:accountAddress' component={Tokens} />
-                <Route
-                  path='/whitelist/:accountAddress'
-                  component={Whitelist}
-                />
-                <Route
-                  path='/backup/:accountAddress'
-                  component={BackupAccount}
-                />
-                <Route
-                  path='/send/:tokenAddress/from/:accountAddress'
-                  component={Send}
-                />
-                <Redirect from='*' to='/' />
-              </Switch>
-            </Router>
+            <Modal
+              title='New version available'
+              description={`${this.state.newRelease.name &&
+                this.state.newRelease.name} was released!`}
+              visible={this.state.newRelease && !this.state.newRelease.ignore}
```

Comment:
consider adding `const { newRelease } = this.state;` to the render method, and then changing the value of this line to `newRelease && !newRelease.ignore`